components/src/core/drawer/drawer-body/nav-item/drawer-nav-item.component.ts

```diff
@@ -79,9 +81,21 @@ export type ActiveItemBackgroundShape = 'round' | 'square';
                     >
                 </div>
             </pxb-info-list-item>
+            <ng-container *ngIf="isRail()">
+                <div
+                    class="pxb-drawer-nav-item-rail square"
```

Comment:
Do we want to make 'square' a pxblue-style class name?

Reply:
Some of our classes are more geared for conditional styling based around internal state.  Originally I was thinking that end users wouldn't be concerned with _every_ class we manage, but it's hard for us to judge which ones to expose.

I think we can make our lives easier by just applying to the `pxb-[component-name]` prefix to every class.  I would make this a separate github issue though because there are a few of these scattered throughout our comp lib.